test.py

```diff
@@ -160,8 +160,16 @@ def spreadTest(generator, expected):
         # sort for easy comparison
         actual[j] = ''.join(sorted(actual[j]))
 
-    assert arraysEqual(expected, actual), "In a large sample of generated slugids, the range of characters found per character position in the sample did not match expected results.\n\nExpected: " + str(expected) + "\n\nActual: " + str(actual)
+    arraysEqual(expected, actual), "In a large sample of generated slugids, the range of characters found per character position in the sample did not match expected results.\n\nExpected: " + str(expected) + "\n\nActual: " + str(actual)
```

Comment:
arraysEqual no longer returns a boolean, so I'm not sure how this text gets used.

* What is the motivation of the change here? Is it to get more granular information about what the difference is between the two arrays?
* The previous implementation `return cmp(a, b) == 0` was pretty lightweight - did you find this inadequate, or is it related to python 3 compatibility?